engine/apps/telegram/tests/test_keyboard_renderer.py

```diff
@@ -49,27 +49,27 @@ def test_actions_keyboard_alerting(make_organization, make_alert_receive_channel
         [
             InlineKeyboardButton(
                 text="Acknowledge",
-                callback_data=f"{alert_group.pk}:acknowledge:oncall-uuid{organization.uuid}",
+                callback_data=f"{alert_group.pk}:0:oncall-uuid{organization.uuid}",
```

Comment:
should these be pulled from `ACTION_TO_CODE_MAP`, rather than hardcoding them here?

Reply:
I hardcoded it's intentionally, to make tests more strict and be protected from changes in the ACTION_TO_CODE_MAP

Reply:
ohhh sorry.. this is tests, not the actual implementation, my bad